CMakeLists.txt

```diff
@@ -62,12 +62,12 @@ set(CMAKE_BUILD_TYPE ${CMAKE_BUILD_TYPE} CACHE STRING "${build_types}" FORCE)
 set_property(CACHE CMAKE_BUILD_TYPE PROPERTY STRINGS "${build_types}")
 
 # Standard must be set both this way and in the flags below, to cover all cmake versions
```

Comment:
Hm. What CMake versions *don't* support `CMAKE_CXX_STANDARD`?

Reply:
Woot! [This](https://crascit.com/2015/03/28/enabling-cxx11-in-cmake/) indicates that CMake 3.8 introduced `CMAKE_CXX_STANDARD 17` support, and 18.04 has CMake 3.10. Looks like this comment is outdated.